kratos/geometries/bounding_box.h

```diff
@@ -167,6 +167,22 @@ class BoundingBox
         }
     }
 
+    /**
+     * @brief Checks if a point is inside the bounding box.
+     * @details This function checks if a given point is inside the bounding box defined by the
+     * minimum and maximum points. It returns true if the point is inside the bounding box,
+     * and false otherwise.
+     * @param rPoint The point to be checked.
+     * @return True if the point is inside the bounding box, false otherwise.
+     */
+    bool IsInside(const array_1d<double, 3>& rPoint) const
```

Comment:
I would add a tolerance here (like the Geometry::IsInside)

Reply:
Done